locales/en/app.ftl

```diff
@@ -854,14 +854,16 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 -brand-firefox = Firefox
 -brand-monitor = Monitor
--brand-fx-monitor = Firefox Monitor
+-brand-fx-monitor = Mozilla Monitor
 -brand-mozilla = Mozilla
 -brand-premium = Premium
```

Comment:
I'm a bit unclear here. Looks like we have about 66 references to a `-brand-premium`:

```sh
git grep "\-brand-premium" | wc -l # 66
```

A bit of lovely inverse grepping shows all instances are either the localized strings in app.ftl, or the one usage of "settings-cancel-premium-subscription-title":
```sh
git grep "\-brand-premium" | grep -v "settings-cancel-premium-subscription-title ="
```

&hellip; along with its one usage in src/views/partials/settings.js:
```sh
git grep "settings-cancel-premium-subscription-title" | cat

locales/cs/settings.ftl:settings-cancel-premium-subscription-title = Zrušit předplatné { -brand-premium }
...
locales/zh-CN/settings.ftl:settings-cancel-premium-subscription-title = 取消 { -brand-premium } 订阅
locales/zh-TW/settings.ftl:settings-cancel-premium-subscription-title = 取消 { -brand-premium } 訂閱
src/views/partials/settings.js:      ${getMessage('settings-cancel-premium-subscription-title')}
```

Does that string need to be tweaked to say "Plus" instead of "Premium" as well?

Reply:
I believe `src/views/partials/settings.js` is unused, the current live version is in `src/app/(nextjs_migration)/(authenticated)/user/settings`.PR #3936

Reply:
I think we should ignore this for now and clean it up along with dead code later, re-open if you disagree.